src/app/modules/item/task/rxjschannel.ts

```diff
@@ -0,0 +1,75 @@
+import { build, ChannelConfiguration, MessageTransaction, MessagingChannel } from 'jschannel';
+import { Observable } from 'rxjs';
+
+export interface RxMessage<T> {
```

Comment:
For the current usage (there is no use of "notify" for now), the generic type is useless here as you do not care about the type of your arg, so you could set it`unknown` and remove the `<T>`.

Reply:
Removed the generic type.